.github/workflows/ci.yml

```diff
@@ -0,0 +1,165 @@
+    needs:
+      - pr
+    runs-on: ubuntu-latest
+    env:
+      ARTIFACT_LOCAL_LOCATION: /opt/artifact
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: ${{ needs.pr.outputs.repro-ci-artifact-name }}
+          path: ${{ env.ARTIFACT_LOCAL_LOCATION }}
+
+      - name: Diff docs/
+        id: diff
+        run: |
+          mv ${{ env.ARTIFACT_LOCAL_LOCATION }}/output000/* ./docs
+          git add .
+          if git diff --cached --exit-code ./docs; then
+            echo "::warning::Modification to 'docs/', need to commit with '!update docs'"
+            exit 1
+          fi
+
+      - name: Diff docs/ notifier
+        if: failure() && steps.diff.outcome == 'failure'
+        env:
+          REPO_TARGET_DOCS_URL: https://${{ github.server_url }}/${{ github.repository}}/tree/${{ github.head_ref }}/docs
+          REPO_SOURCE_ARTIFACT_URL: ${{ needs.pr.outputs.repro-ci-artifact-url }}
+        uses: access-nri/actions/.github/actions/pr-comment@main
+        with:
+          pr: ${{ github.event.issue.number }}
+          comment: |
+            The `docs` in ${{ env.REPO_TARGET_DOCS_URL }} differ from ${{ env.REPO_SOURCE_ARTIFACT_URL }} (in directory `output000`).
+
+            Comment `!update_docs` if you want these changes committed.
+
+  ###################
+  # PR Comment jobs #
+  ###################
+  pr-comment:
+    name: Comment
+    if: github.event_name == 'issue_comment' && github.repository != 'ACCESS-NRI/model-configs-template'
+    uses: access-nri/model-config-tests/.github/workflows/config-pr-2-confirm.yml@main
+    secrets: inherit
+    permissions:
+      contents: write  # For updating metadata.yaml version and committing checksums
+      pull-requests: write  # For commenting on PR
+
+  pr-comment-docs:
+    name: Comment Docs
+    if: github.event_name == 'issue_comment' && github.event.comment.body == '!update_docs' && github.repository != 'ACCESS-NRI/model-configs-template'
+    runs-on: ubuntu-latest
+    env:
+      ARTIFACT_LOCAL_LOCATION: /opt/artifact
+      GH_TOKEN: ${{ github.token }}
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: access-nri/actions/.github/actions/react-to-comment@main
+        with:
+          reaction: rocket
+          token: ${{ github.token }}
+
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.GH_COMMIT_CHECK_TOKEN }}
+
+      - name: Checkout Associated PR ${{ github.event.issue.number }}
+        # Since the trigger for this workflow was on.issue_comment, we need
+        # to do a bit more wrangling to checkout the pull request and get the branch name
+        id: pr
+        run: |
+          gh pr checkout ${{ github.event.issue.number }}
+          echo "branch=$(git rev-parse --abbrev-ref HEAD)" >> $GITHUB_OUTPUT
+
+      - name: Download Newly Created Checksum
```

Comment:
Is this name correct? I think this is downloading the newly created docs. No?

Reply:
Whoops...similar bit of logic in `model-config-tests` that I hadn't changed the name of

Reply:
Done in https://github.com/ACCESS-NRI/access-om3-configs/pull/122/commits/10b88ec3b8b1ca780483314e61a53572e6efd90e